src/workers.ts

```diff
@@ -40,37 +42,29 @@ if (!view.document) {
 /** Whether or not sombra has been initialized */
 const initialized = false;
 
-// /////// //
-// Helpers //
-// /////// //
-
-const scriptElement: HTMLScriptElement | undefined | null =
-  document.currentScript ||
-  (document.querySelector('script[data-penumbra]') as any);
+// //////////// //
+// Load Workers //
+// //////////// //
 
-/**
- * Get the script element and throw an error if it cannot be found on the DOM
- */
-function getScriptElement(): HTMLScriptElement {
-  if (!scriptElement) {
-    throw new Error('Unable to locate Penumbra script element.');
-  }
-  return scriptElement;
+if (SHOULD_LOG_EVENTS) {
+  console.info('Loading penumbra script element...');
+}
+let scriptElement: HTMLScriptElement = (document.currentScript ||
+  document.querySelector('script[data-penumbra]')) as any;
+if (!scriptElement && SHOULD_LOG_EVENTS) {
+  scriptElement = { dataset: {} } as any;
```

Comment:
I think the `scriptElement = { dataset: {} } as any` should be outside fo this if statement so that it can still be assigned when `SHOULD_LOG_EVENTS === false`.